### DISCUSSION TITLE
Welcome to 3Webpages Discussions!

### DISCUSSION THREAD

**Discussion by pjjj23** [pjjj23/3Webpages#1] [Announcements]
## 👋 **Welcome!**
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  _To get started, comment below with an introduction of yourself and tell us about what you do with this community._

**Comment by pjjj23** [comment]
hello, please leave a comment. Thanks!

**Comment by datalinkscrummaster** [comment]
hello
